config/sync/core.entity_view_display.node.stanford_news.search_indexing.yml

```diff
@@ -126,9 +128,11 @@ hidden:
   links: true
   search_api_excerpt: true
   stanford_intranet__access: true
+  su_metatags: true
   su_news_dek: true
   su_sdss_magazine_topics: true
   su_sdss_news_focal_areas: true
+  su_sdss_news_news_source: true
```

Comment:
no to being indexed?

Reply:
Good question. I'll ask in the ticket. This is just adding the field for data entry and there will be later iterations.